pyproject.toml

```diff
@@ -33,6 +33,7 @@ dependencies = [
     "scikit-learn>=0.21, <=1.4.2",
     "scipy>=1.2.0, <=1.13.0",
     "threadpoolctl",
+    "tqdm",
```

Comment:
Do we really need `tqdm`? I just want to make sure we don't add a dependency that isn't necessary (not saying it isn't in this case).

Reply:
This is a baked-in dependency for robustica so I guess the idea is, once we start using it, we can use it. That said, the place it is used in the code is a really inefficient voxel-wise for loop. There's a way to make that for loop a lot more efficient (doing the calculation on multiple voxels for each number of echoes) so we should be able to remove the need to actually use tqdm in this PR.